### DISCUSSION TITLE
Best practice Teufel Raumfeld speakers

### DISCUSSION THREAD

**Discussion by riesi77** [music-assistant/support#1640] [Q&A] [answered]
Hi,
I have 2 Teufel Raumfeld speakers: A "M" 1st Gen called "Wohnzimmer" and a "S" 2nd Gen called "Waldzimmer".
In Music Assistant (2.0) I get them available via DLNA/uPNP, but I see multiple entries which I don't understand:
![image](https://github.com/music-assistant/hass-music-assistant/assets/67387975/e65e7c81-2ae1-4733-84ff-06ab39dc2a35)

Can someone explain which speaker to use and which one to disable?

On Top: In Home Assistant, I have several entries because of the dlna integration and several because of the Music Assistant Integration. Would you recommend to disable the dlna Speakers?

Thanks - also for the great work on Music Assistant 2.0!

**Comment by OzGav** [comment, marked as answer]
In your image all but the first one is showing as unavailable. DLNA is a poorly implemented protocol by many manufacturers. Marcel will no doubt have more of an idea but I would disable any duplicates. If you can find one that works that is all you need.
